queries.sql

```diff
@@ -8,3 +8,33 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = TRUE;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE 10.4 <= weight_kg AND weight_kg >= 17.3;
+
+BEGIN;
+ALTER TABLE animals RENAME COLUMN species TO unspecified;
+SELECT * FROM animals;
```

Comment:
- [ ] After making species `unspecified` you are supposed to verify that change was made
```sql
SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

Reply:
Yeah yeah! I have forgotten to put all those lines in the repos.